### PR TITLE
Add some flags from newer libcryptsetup versions

### PR DESCRIPTION
**Pull request by refi64**
This particularly adds the NO_READ/WRITE_WORKQUEUE flags.

### CODE REVIEW

**Comment by refi64** [comment]
Okay the latest version should work on older versions.

File: src/consts/flags.rs [hunk 1]
```diff
@@ -28,6 +28,16 @@ bitflags! {
...
+        #[cfg(cryptsetup23supported)]
+        const SUSPENDED = libcryptsetup_rs_sys::crypt_activate_suspended;
+        #[cfg(cryptsetup24supported)]
+        const IV_ACTIVATE_LARGE_SECTORS = libcryptsetup_rs_sys::crypt_activate_iv_large_sectors;
+        #[cfg(cryptsetup24supported)]
+        const PANIC_ON_CORRUPTION = libcryptsetup_rs_sys::crypt_activate_panic_on_corruption;
+        #[cfg(cryptsetup24supported)]
+        const NO_READ_WORKQUEUE = libcryptsetup_rs_sys::crypt_activate_no_read_workqueue;
+        #[cfg(cryptsetup24supported)]
+        const NO_WRITE_WORKQUEUE = libcryptsetup_rs_sys::crypt_activate_no_write_workqueue;
```
Comment (jbaublitz) [inline src/consts/flags.rs:40]: Would you also mind adding `RECALCULATE_RESET` here flagged as `cryptsetup24supported`? Looking at the header, it looks like that was added too.

**Review by jbaublitz — CHANGES_REQUESTED** [review]
Just a couple of things that I'd like to change, but overall this looks good. The feature flagging all looks like it should work based on looking at the cryptsetup 2.3.0 vs 2.4.0 headers. Thanks for doing this!

File: src/consts/flags.rs [hunk 2]
```diff
@@ -28,6 +28,16 @@ bitflags! {
         const REFRESH = libcryptsetup_rs_sys::crypt_activate_refresh;
         const SERIALIZE_MEMORY_HARD_PBKDF = libcryptsetup_rs_sys::crypt_activate_serialize_memory_hard_pbkdf;
         const NO_JOURNAL_BITMAP = libcryptsetup_rs_sys::crypt_activate_no_journal_bitmap;
+        #[cfg(cryptsetup23supported)]
+        const SUSPENDED = libcryptsetup_rs_sys::crypt_activate_suspended;
+        #[cfg(cryptsetup24supported)]
+        const IV_ACTIVATE_LARGE_SECTORS = libcryptsetup_rs_sys::crypt_activate_iv_large_sectors;
```
[the remark this answers is not in the document]
Comment (refi64) [reply src/consts/flags.rs]: oops, ngl not sure why I did that :sweat_smile:

**Review by jbaublitz — APPROVED** [review]

**Comment by jbaublitz** [comment]
This looks great! Thanks for the contribution.

**Pull request merged**